pkg/nsx/services/staticroute/staticroute.go

```diff
@@ -92,7 +92,7 @@ func (service *StaticRouteService) CreateOrUpdateStaticRoute(namespace string, o
 
 	vpc := service.VPCService.ListVPCInfo(namespace)
 	if len(vpc) == 0 {
-		return fmt.Errorf("no vpc found for ns %s", namespace)
+		return fmt.Errorf("no vpc found for ns %s, will retry later", namespace)
```

Comment:
-> no VPC found for namespace
only here uses 'ns'

Reply:
updated, thanks!